content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -14,7 +14,7 @@ weight: 30
 {{< feature-state for_k8s_version="v1.16" state="beta" >}}
 This document describes the concept of cloning existing CSI Volumes in Kubernetes.  Familiarity with [Volumes](/docs/concepts/storage/volumes) is suggested.
 
-This feature requires VolumePVCDataSource feature gate to be enabled:
+This feature requires VolumePVCDataSource feature gate to be enabled (it is by default):
```

Comment:
I think we can actually remove these 2 lines about feature gates since in beta it is on by default.

Reply:
Thanks for the review, and  i will remove these.